week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,17 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+// It calls a function to return a number by generatingrandon number between 0 and 1
+// and multiply it with 10.Then it logs the result .
 // - How many parameters does the function take?
+//  There is no parameters for this function
 // - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+//   Function name is getNumber
+// - Where is the function called? with what parameters?
+//  Function is called on line 5 var result = getNumber()
```

Comment:
Here I'd say: It's called on line 5 with no parameters.

Reply:
Excellent!

Reply:
will keep that in mind